### PR TITLE
add DMS sample app to developer hub

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: data/developerhub/applications.json [hunk 1]
```diff
@@ -376,6 +376,19 @@
       "complexity": ["basic"],
       "pro": true,
       "cloudpods": false
+    },
+    {
+      "title": "Deploying a Database Migration Pipeline and creating CDC with DMS, Kinesis, and RDS MariaDB",
```
Comment (cloutierMat) [inline data/developerhub/applications.json:381]: Is there a reason for the emphasis on CDC in the title?

**Review by cloutierMat — APPROVED** [review]
:rocket:
Looks good to me. Just left one question. :smile:

File: data/developerhub/applications.json (hunk 1 above)
Comment (HarshCasper) [reply data/developerhub/applications.json:381]: None. Let me remove that :)

Comment (cloutierMat) [reply data/developerhub/applications.json]: I don't know if it rolls of the tongue as easily, but might be more accurate! :smile: 
```suggestion
      "title": "Deploying a Database Migration Pipeline with DMS, creating Replication Tasks from RDS MariaDB to Kinesis",
```

**Pull request merged**